src/meta/src/hummock/manager/table_write_throughput_statistic.rs

```diff
@@ -26,6 +26,12 @@ impl AsRef<TableWriteThroughputStatistic> for TableWriteThroughputStatistic {
     }
 }
 
+impl TableWriteThroughputStatistic {
+    pub fn is_expired(&self, max_statistic_expired_secs: i64, timestamp_secs: i64) -> bool {
+        timestamp_secs - self.timestamp_secs > max_statistic_expired_secs
```

Comment:
minor: how about using `(timestamp_secs - self.timestamp_sec).max(0)` to avoid surprises